media-collections/add-and-remove-assets-from-media-collections.md

```diff
@@ -0,0 +1,38 @@
+# Add and remove assets from media collection
+
+You can simply add or remove files and folders from media collections. Adding an asset to a media collection places it in a virtual group without redundancy while keeping the actual asset unaffected by any change in location or URL. Removing the asset simply removes the above association with the asset. Using this workflow makes it easy for an administrator to manage and share resources with users.
+
+Learn more about permissions for these operations for restricted users [here](../access-control-and-permissions/README.md#permissions).
+
+## Add assets to media collections
+
+In the ImageKit media library, select and right-click the files and folders you want to add to media collections and click on the "Add to media collections" option. You can only add assets if you have at least "contribute" permission to a media collection. A popup window would appear, allowing you to search for and select the media collections to which you wanted to add the assets. Only the media collections you can "manage" will appear in the list. After selecting the media collections, click on "Submit." 
```

Comment:
> Only the media collections you can "manage" will appear in the list. After selecting the media collections, click on "Submit." 

Adding assets is allowed on contribute access

Reply:
done